src/app/system/global-configurations/edit-configuration/edit-configuration.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="configurationForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Configuration Value</mat-label>
+            <input matInput required formControlName="value" [(ngModel)]="configurationData.value">
+            <mat-error *ngIf="configurationForm.controls.value.hasError('required')">
+              Configuration Value is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+        </div>
+
+      </mat-card-content>
+
+      <mat-card-actions fxLayout="row" fxLayout.xs="column" fxLayoutAlign="center" fxLayoutGap="5px">
+        <button type="button" mat-raised-button [routerLink]="['../../']"> Cancel </button>
+        <button mat-raised-button color="primary" [disabled]="!configurationForm.valid"> Submit </button>
```

Comment:
I don't think so because one use-case can be that user comes to edit the value but it sees that it is already alright then clicks submit or another use-case can be that user accidentally clicks the Edit button and then want to go to the configurations list again.

Reply:
That's what the cancel button is for.